### DISCUSSION TITLE
Display Issue

### DISCUSSION THREAD

**Discussion by d0nizam** [coobbi/Jre4android#4] [General]
Dear JRE4Android Support Team,

I hope this message finds you well.I am encountering a display issue with the JRE4Android app (version 1.6.30) on my Samsung Galaxy A54 5G running Android 14.

 In the app’s settings, the display is entirely white until I switch from dark mode to light mode in my Samsung device settings. This issue affects the visibility of the settings options.

Here are the details:App Version: 1.6.30Device Model: Samsung Galaxy A54 5GAndroid Version: 14

**Comment by d0nizam** [comment]
![1000030147](https://github.com/user-attachments/assets/8370c9d2-929b-4383-856b-a281d3f5d355)

**Reply by coobbi** [reply]
It was fixed, you can update it at google play now

**Comment by coobbi** [comment]
it is a bug, it will be fixed at 1.6.31
